### PR TITLE
Homework13

### PR DESCRIPTION
**Pull request by bodenkovv**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/BodenkoVV/AskQuestion/Observer/Catalog/Block/Product/View.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+namespace PalashAs\AskQuestion\Observer\Catalog\Block\Product;
```
Comment (chabanenk0) [inline app/code/BodenkoVV/AskQuestion/Observer/Catalog/Block/Product/View.php]: namespace BodenkoVV\AskQuestion\Observer\Catalog\Block\Product;
?

File: app/code/BodenkoVV/AskQuestion/Observer/Catalog/Block/Product/View.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        $this->_registry = $registry;
+    }
+
+    /**
+     * @param Observer $observer
+     * @return $this
+     */
+    public function execute(Observer $observer)
+    {
+        $event = $observer->getEvent();
```
Comment (chabanenk0) [inline app/code/BodenkoVV/AskQuestion/Observer/Catalog/Block/Product/View.php]: Was unable to found where this observer is attached to any of frontend event. Currently the form is shown on product page regardless if the attribute allow_to_ask_questions has value "yes" or "no".Please re-check this event and its observer

Comment (bodenkovv) [reply app/code/BodenkoVV/AskQuestion/Observer/Catalog/Block/Product/View.php]: +

**Comment by chabanenk0** [comment]
@bodenkovv  Currently product page doesn't work:
The file BodenkoVV\AskQuestion\Observer\Catalog\Block\Product\View is removed, however this observer is used here: https://github.com/bodenkovv/magento226/blob/homework13/app/code/BodenkoVV/AskQuestion/etc/frontend/events.xml#L5

This observer should be  resonsive for request form display control, so you shouldn't remove it, please fix its functionality

**Pull request merged**